samcli/local/docker/lambda_image.py

```diff
@@ -37,6 +37,7 @@ class Runtime(Enum):
     go1x = "go1.x"
     dotnetcore21 = "dotnetcore2.1"
     dotnetcore31 = "dotnetcore3.1"
+    dotnet5 = "dotnet5.0"
```

Comment:
Is this needed for something? 

Off the top of my head, this enum is heavily used for Zips but not for images (or wouldn't expect it to). This is probably fine but not sure what this may affect.

Reply:
I think this maybe because and these are used keys into the manifest.json from app templates. we need to decouple this.

Reply:
You are right, this is not needed anymore, tested with guided `init` and non-interactive one(`samdev init --no-interactive --name dotnet5-18-1738 --package-type Image --base-image amazon/dotnet5.0-base`) I was able to initiate the project.